pkgs/applications/audio/CHOWTapeModel/default.nix

```diff
@@ -41,36 +76,51 @@ stdenv.mkDerivation rec {
     libsepol
     libsysprof-capture
     libthai
+    libuuid
     libxkbcommon
     lv2
     pcre
+    pcre2
```

Comment:
Do we need both pcre and pcre2?

Reply:
Good question, I went after what the configuration tool reported as missing. When I remove pcre2 it says:

```
Package libpcre2-8 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libpcre2-8.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libpcre2-8', required by 'glib-2.0', not found
```

When I remove `pcre`, I get:

```
Package libpcre was not found in the pkg-config search path.
Perhaps you should add the directory containing `libpcre.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libpcre', required by 'libselinux', not found
```

I haven't checked if they are truly required or if the build succeeds without, though,

Reply:
OK, thanks, looks like they are needed then.

Reply:
OK, thanks, looks like they are needed then.